src/auth/appConfig.ts

```diff
@@ -10,28 +10,34 @@ import { DEFAULT_CORE_NODE, DEFAULT_SCOPE, DEFAULT_BLOCKSTACK_HOST } from './aut
  * `window.location.origin` as the app domain.
  * On non-browser platforms, you need to
  * specify an app domain as the first argument.
+ * 
+ * 
  * @type {AppConfig}
```

Comment:
All of these `@type {...}` are jsdoc/flow leftovers, we can remove them so they don't show up as generic/unknown tags in the typedoc gen